content/html/concepts/semantic-html/terms/section/section.md

```diff
@@ -34,7 +35,96 @@ In the following code, we created a `<section>` element to encapsulate the code.
   </article>
 </section>
 ```
-
+  
+There are also examples where we can use other elements that semantically fit better then `<section>` element.
+  
+- The first example is the `<header>` element which can be used to describe that the content inside is the header of the page.
```

Comment:
Because of the interspersed code blocks, this doesn't really work as a bulleted list. I'd remove the bullets and have the code blocks introduced with regular paragraphs.